docs/mkdocs.yml

```diff
@@ -28,6 +28,7 @@ nav:
           - Nextcloud: guides/auth/nextcloud.md
           - Twitter: guides/auth/twitter.md
           - Authelia: guides/auth/authelia.md
+          - Azure Active Directory: guides/auth/azure-md.md
```

Comment:
wrong file name

Reply:
I am very sorry for my clumsiness. I have completed the requirements. By mistake my lsp converted the single quotes to double quotes but I have fixed it. If it will affect your commit history may be I can start over and commit and pull request from the start? It's my first pr and I am feeling quite anxious